src/components/Picker-group/Picker-group.js

```diff
@@ -81,7 +81,7 @@ export default class PickerGroup extends React.Component {
       <div ref={this.props.uxpinRef} style={{minWidth: 308}}>
         <fieldset>
```

Comment:
As you can see the picker group doesn't look right, you will need to add a `min-width` when you enable the `fluid` option to make it look right and not break.

![image](https://user-images.githubusercontent.com/86958774/175922575-bda40679-da64-412b-aac7-27d73dcecee8.png)
```suggestion
      <fieldset ref={this.props.uxpinRef} style={{ minWidth: this.props.fluid ? 310 : 308 }}>
```